### PR TITLE
add a file with TAC people & their roles

### PR DESCRIPTION
**Pull request by thomas-fossati**
Also, add links in the github process description that point to that file (and CODEOWNERS too).

Note that I couldn't find a list of TAC members apart from the voting reps.  The only (semi-official) source seems to be the CODEOWNERS file, but I'm not sure how accurate that is?

### CODE REVIEW

**Review by dthaler — APPROVED** [review]

**Comment by dthaler** [comment]
Adding time bomb label to expire Feb. 15 per https://github.com/confidential-computing/governance/blob/main/tac-github-process.md

File: tac-composition.md
Comment (lkatalin) [inline tac-composition.md:17]:
```suggestion
   * Lily Sturmann (@lkatalin), Red Hat
```
smallest nit possible : )

**Review by lkatalin — APPROVED** [review]

File: tac-github-process.md
Comment (dcmiddle) [inline tac-github-process.md:7]: Probably better not to link to CODEOWNERS here since you have added the better file (tac-composition.md). Currently CODEOWNERS seems to have non-TAC members, but maybe that's in error.
If you want to connect to CODEOWNERS maybe link it to `reviewers` 2 lines below.

**Review by dcmiddle — CHANGES_REQUESTED** [review]

File: tac-github-process.md
Comment (thomas-fossati) [reply tac-github-process.md:7]: My plan A was I'd list all TAC members (not just the voting reps) in the new file.  The thing is I couldn't find an "official" & up-to-date list of *all* TAC members anywhere, so CODEOWNERS seemed like a decent approximation.

Do you know whether such list exists somewhere?  If so, I could amend `tac-composition.md` and change the link in `tac-github-process.md` accordingly.

Comment (dcmiddle) [reply tac-github-process.md:7]: I see. I don't what the original text about members referred to. I think the only formalism is the voting members. Other contributors can probably opt-in to tracking the repo with the `watch` feature.

Comment (dcmiddle) [reply tac-github-process.md:7]: If we're going to define members and voting members then line 10 needs to be updated to read 'Once PRs are approved by 2 TAC voting members...'

Comment (dcmiddle) [reply tac-github-process.md:7]: I think it's more pragmatic, though, to only maintain a list of voting members in this repo and let any other contributors, participants, etc. use the github watch feature.

**Comment by brianwarner** [comment]
Would there be any objections if I add something here about staff pushing/merging trivial changes directly? For example, when I'm uploading a PDF of someone's presentation, or a meeting minutes template?

File: tac-composition.md
Comment (dthaler) [inline tac-composition.md:8]: Need to update this to be Kurt, and line 11 to be Tate

**Comment by dthaler** [comment]
@thomas-fossati can you update this PR?  thanks!

File: tac-composition.md
Comment (dcmiddle) [inline tac-composition.md:5]: We did not IIRC establish a vice chair this term.

File: tac-composition.md
Comment (lkatalin) [inline tac-composition.md:16]:
```suggestion
   * Lily Sturmann (@lkatalin) and Yash Mankad (@yashkmankad ), Red Hat
```

**Review by lkatalin — CHANGES_REQUESTED** [review]
Please see inline about adding Yash as co-rep.

File: tac-composition.md
Comment (thomas-fossati) [inline tac-composition.md:6]:
```suggestion
```

**Review by dcmiddle — APPROVED** [review]
thanks @thomas-fossati

**Comment by thomas-fossati** [comment]
@lkatalin: please check that everything is in good shape.  If so, feel free to merge.

**Pull request merged**
